### PR TITLE
VladDmytriv - add test to cover OrderingPage functionality and POs methods for this test

### PR DESCRIPTION
**Pull request by vladdmytriv**

### CODE REVIEW

File: src/main/java/com/ss/ita/rozetka/ui/pages/OrderingPage.java [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
-    //For writing methods in close future
+
+    @Step("OrderingPage: get total products price")
+    public int getTotalProductsPrice() {
+        String price = ($x("//dd[@class = 'checkout-total__value checkout-total__value_size_large']").text());
+        return Integer.parseInt(price.replaceAll("[^0-9]", StringUtils.EMPTY));
+    }
+
+    @Step("OrderingPage: verify that OrderingPage is opened")
+    public OrderingPage sidebarShouldBeVisible() {
```
Comment (toleksyn) [inline src/main/java/com/ss/ita/rozetka/ui/pages/OrderingPage.java:18]: get status, assert in test

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/main/java/com/ss/ita/rozetka/ui/pages/ProductPage.java [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
     @Step("ProductPage: add product to basket")
     public BasketModal addProductToBasket() {
-        $x("//button[contains(@class,'buy-button button button_')]").click();
+        actions().moveToElement($x("//button[contains(@class,'buy-button button button_')]")).click().perform();
```
Comment (toleksyn) [inline src/main/java/com/ss/ita/rozetka/ui/pages/ProductPage.java:12]: format

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/main/java/com/ss/ita/rozetka/ui/pages/OrderingPage.java [hunk 3]
```diff
@@ -1,5 +1,29 @@
...
+
+    @Step("OrderingPage: get total products price")
+    public int getTotalProductsPrice() {
+        return parseInt($x("//dd[@class = 'checkout-total__value checkout-total__value_size_large']")
+                .text()
+                .replaceAll("[^0-9]", StringUtils.EMPTY));
+    }
+
+    @Step("OrderingPage: get ordering page status, assert in test")
+    public OrderingPage orderingPageShouldBeOpened() {
```
Comment (toleksyn) [inline src/main/java/com/ss/ita/rozetka/ui/pages/OrderingPage.java:20]: boolean isOrderingPageOpened()

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/main/java/com/ss/ita/rozetka/pageobject/pages/OrderingPage.java [hunk 4]
```diff
@@ -1,5 +1,29 @@
...
+
+    @Step("OrderingPage: get total products price")
+    public int getTotalProductsPrice() {
+        return parseInt($x("//dd[@class = 'checkout-total__value checkout-total__value_size_large']")
+                .text()
+                .replaceAll("[^0-9]", StringUtils.EMPTY));
+    }
+
+    @Step("OrderingPage: get ordering page status, assert in test")
+    public OrderingPage orderingPageShouldBeOpened() {
```
Comment (toleksyn) [inline src/main/java/com/ss/ita/rozetka/pageobject/pages/OrderingPage.java:20]: re-do method to boolean isOpened() and assert in test 
fix step

**Review by toleksyn — CHANGES_REQUESTED** [review]

**Review by Percifalll — APPROVED** [review]

File: src/test/java/com/ss/ita/rozetka/test/OrderingProductsTest.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+        int productPrice = productPage.getPrice();
+        String productTitle = productPage.getName();//should be renamed by another PR
+
+        OrderingPage orderingPage = productPage
+                .addProductToBasket()
+                .orderProducts();
+
+        assertThat(orderingPage.isOrderingPageOpened())
+                .as("OpderingPage should be opened")
```
Comment (toleksyn) [inline src/test/java/com/ss/ita/rozetka/test/OrderingProductsTest.java:31]: fix

File: src/main/java/com/ss/ita/rozetka/pageobject/pages/OrderingPage.java [hunk 6]
```diff
@@ -1,5 +1,30 @@
...
-    //For writing methods in close future
+
+    @Step("OrderingPage: get total products price")
+    public int getTotalProductsPrice() {
+        return parseInt($x("//dd[@class = 'checkout-total__value checkout-total__value_size_large']")
+                .text()
+                .replaceAll("[^0-9]", StringUtils.EMPTY));
+    }
+
+    @Step("OrderingPage: get ordering page status, assert in test")
```
Comment (toleksyn) [inline src/main/java/com/ss/ita/rozetka/pageobject/pages/OrderingPage.java:19]: remove assert in test

**Review by toleksyn — CHANGES_REQUESTED** [review]

**Pull request merged**